mozilla_vpn/dashboards/mozilla_vpn.dashboard.lookml

```diff
@@ -410,36 +418,58 @@
     col: 12
     width: 12
     height: 9
-  - title: 7-day Average of Retention Rate
-    name: 7-day Average of Retention Rate
+  - title: Churn Rate 7-day Average
+    name: Churn Rate 7-day Average
```

Comment:
retention dashboards define retention differently than this chart did, so now it shows churn for consistency.